cpu/native/include/socket_zep.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    netdev_socket_zep  ZEP-based IEEE 802.15.4 device for native
+ * @ingroup     netdev
+ * @brief
```

Comment:
mhm, a bit sad this is empty 😢 Maybe put the (rather long) defgroup title here, and shorten the above?